__init__.py

```diff
@@ -1,25 +1,17 @@
 from fastapi import APIRouter
 
 from lnbits.db import Database
-from lnbits.helpers import template_renderer
+from .views_api import lndhub_api_router
+from .views import lndhub_generic_router
 
 db = Database("ext_lndhub")
-
 lndhub_ext: APIRouter = APIRouter(prefix="/lndhub", tags=["lndhub"])
-
 lndhub_static_files = [
     {
         "path": "/lndhub/static",
         "name": "lndhub_static",
     }
 ]
 
-
-def lndhub_renderer():
```

Comment:
in all other extensions the `_renderer()` function is in `__init__`

Reply:
which makes it a circular import, its only needed in generic view though